CMakeSettings.json

```diff
@@ -1,349 +1,365 @@
-            "value": "ON"
-          },
-          {
-            "name": "LLVM_DEFAULT_TARGET_TRIPLE:STRING",
-            "value": "dxil-ms-dx"
-          },
-          {
-            "name": "CLANG_BUILD_EXAMPLES:BOOL",
-            "value": "OFF"
-          },
-          {
-            "name": "-DLLVM_REQUIRES_RTTI:BOOL",
-            "value": "ON"
-          },
-          {
-            "name": "-DCLANG_CL:BOOL",
-            "value": "OFF"
-          },
-          {
-            "name": "-DCMAKE_SYSTEM_VERSION",
-            "value": "10.0.14393.0"
-          }
-        ]
-        },
-        {
-        "name": "x64-Release",
-        "generator": "Visual Studio 15 2017 Win64",
-        "configurationType" : "Release",
-        "buildRoot":  "${env.LOCALAPPDATA}\\CMakeBuild\\DirectXShaderCompiler\\build\\${name}",
-        "cmakeCommandArgs":  "",
-        "buildCommandArgs": "-m -v:minimal",
-        "variables": [
-          {
-            "name": "CLANG_ENABLE_ARCMT:BOOL",
-            "value": "OFF"
-          },
-          {
-            "name": "CLANG_ENABLE_STATIC_ANALYZER:BOOL",
-            "value": "OFF"
-          },
-          {
-            "name": "CLANG_INCLUDE_TESTS:BOOL",
-            "value": "OFF"
-          },
-          {
-            "name": "LLVM_INCLUDE_TESTS:BOOL",
-            "value": "OFF"
-          },
-          {
-            "name": "HLSL_INCLUDE_TESTS:BOOL",
-            "value": "ON"
-          },
-          {
-            "name": "LLVM_TARGETS_TO_BUILD:STRING",
-            "value": "None"
-          },
-          {
-            "name": "LLVM_INCLUDE_DOCS:BOOL",
-            "value": "OFF"
-          },
-          {
-            "name": "LLVM_INCLUDE_EXAMPLES:BOOL",
-            "value": "OFF"
-          },
-          {
-            "name": "LIBCLANG_BUILD_STATIC:BOOL",
-            "value": "ON"
-          },
-          {
-            "name": "LLVM_OPTIMIZED_TABLEGEN:BOOL",
-            "value": "OFF"
-          },
-          {
-            "name": "LLVM_REQUIRES_EH:BOOL",
-            "value": "ON"
-          },
-          {
-            "name": "LLVM_APPEND_VC_REV:BOOL",
-            "value": "ON"
-          },
-          {
-            "name": "LLVM_ENABLE_RTTI:BOOL",
-            "value": "ON"
-          },
-          {
-            "name": "LLVM_ENABLE_EH:BOOL",
-            "value": "ON"
-          },
-          {
-            "name": "LLVM_DEFAULT_TARGET_TRIPLE:STRING",
-            "value": "dxil-ms-dx"
-          },
-          {
-            "name": "CLANG_BUILD_EXAMPLES:BOOL",
-            "value": "OFF"
-          },
-          {
-            "name": "-DLLVM_REQUIRES_RTTI:BOOL",
-            "value": "ON"
-          },
-          {
-            "name": "-DCLANG_CL:BOOL",
-            "value": "OFF"
-          },
-          {
-            "name": "-DCMAKE_SYSTEM_VERSION",
-            "value": "10.0.14393.0"
-          }
-        ]
-        }
-    ]
+{

```

Comment:
It is difficult to see what changed here because it flags the whole file as a diff. Is it possible to reduce the diff? From the commit it seems you only modified a few defines.